pkg/apiserver/manager/hwameistor/localstoragenode_controller.go

```diff
@@ -596,25 +597,26 @@ func (lsnController *LocalStorageNodeController) StorageNodePoolDiskGet(page hwa
 	nodeKey := client.ObjectKey{
 		Name: page.NodeName,
 	}
-
+	// get localdisk information
+	devicePath := hwameistorapi.DEV + page.DiskName
+	localDisks, e := lsnController.ldHandler.ListLocalDiskByNodeDevicePath(page.NodeName, devicePath)
+	if e != nil {
+		log.WithField("nodeName", page.NodeName).WithField("devPath", devicePath).WithError(e).Errorf("Failed to get localdisk")
+		return nil, e
+	}
+	if len(localDisks) == 0 {
+		log.WithField("nodeName", page.NodeName).WithField("devPath", devicePath).WithError(e).Errorf("The length of localDisks list to be zero,failed to get localdisk")
```

Comment:
this message has no meaning, may use `no localdisks found` rather use length zero as message

Reply:
Got it!

Reply:
Thanks~~The modified have been applied!